giskard/llm/generators/base.py

```diff
@@ -31,11 +33,16 @@ def __init__(
         llm_temperature: Optional[float] = None,
         llm_client: LLMClient = None,
         prompt: Optional[str] = None,
+        languages: list[str] = None,
     ):
         self.llm_temperature = llm_temperature if llm_temperature is not None else self._default_temperature
         self.llm_client = llm_client or get_default_client()
+        self.languages = languages if languages is not None else []
         self.prompt = prompt if prompt is not None else self._default_prompt
 
+        if isinstance(self.languages, list) and self.languages != []:
+            self.prompt = self.prompt + LANGUAGE_REQUIREMENT_PROMPT.format(languages=self.languages)
```

Comment:
I agree it would be better, but since some of the `Generator` override the `_make_generate_input_prompt` method it will require duplicating this in all these classes. Should I proceed anyway and move it to  `_make_generate_input_prompt` ?